proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+# Cosmos Stargate Hub Upgrade Proposal 2: Chevrons Locked. Time to Upgrade
+
+![Stargate Upgrade banner graphic](https://siasky.net/AADU5rg0GvapbrICLUk7SxZeMDrKo5Q8oLrpOwl71wXhnQ)
+
+## by Iqlusion
+
+Jan 11, 2021
+
+## Key Results
+
+------------
+If passed, this governance proposal would commit the Cosmos Hub to halting the `cosmoshub-3` at 06:00 UTC on Jan 28th exporting the state and starting `cosmoshub-4` based on gaia 3.0.
+
+## Context
+
+------------
+
+In proposal 27, Iqlusion proposed a comprehensive process to translate the unprecented surface area of this upgrade. We called this process Stargate.
+
+Iqlusion is pleased to report that the Stargate Process has been successfully executed. We believe the chevrons are unlocked and that Hub can safely step through the Stargate.
```

Comment:
What is a chevron?

Reply:
Stargate SG1 lore.

Reply:
I'd advocate for using less arcane language here, but it's your call